src/System.Composition.TypedParts/src/System.Composition.TypedParts.csproj

```diff
@@ -0,0 +1,84 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
We standardized on `ToolsVersion="12.0"` in the other projects. This applies to all the .csproj files in this PR.